extensions/data-workspace/src/dialogs/newProjectDialog.ts

```diff
@@ -193,6 +193,15 @@ export class NewProjectDialog extends DialogBase {
 			width: constants.DefaultInputWidth
 		}).component();
 
+		const defaultSaveLocation = defaultProjectSaveLocation();
+		if (defaultSaveLocation) {
+			// use the default save location if it's there
+			locationTextBox.value = defaultSaveLocation.fsPath;
```

Comment:
Could you simplify this to 

`locationTextBox.value = defaultSaveLocation?.fsPath ?? vscode.workspace.workspaceFolders?.[0]?.uri.fsPath;` ? It'll default to undefined if neither of those values exist which is the normal default value anyways.